### PR TITLE
GitHub actions logic to push container to ghcr.io on tag

### PR DESCRIPTION
**Pull request by alexiswl**
Automation workflow to push RNASum container to GitHub container registry for easy automation / building of RNASum Docker container.

Simply create a tag i.e v0.4.2 and then push the tag to remote and watch the magic.

- [x]  Waiting on #45 before merging

### CODE REVIEW

**Comment by alexiswl** [comment]
Holding off on multi-platform for now, qemu / buildx don't seem to be handling installing R on a virtual arm instance on intel architecture.  Will wait until GitHub actions supports arm VMs and can build from a matching host.  Logic is there though once this becomes possible.  

For now you can find the container in [packages](https://github.com/umccr/RNAsum/pkgs/container/rnasum) and test and pull.  

Note that 0.4.2-dev tag is combination of #45, #52 and #54.

**Comment by skanwal** [comment]
@alexiswl - now that https://github.com/umccr/RNAsum/pull/45 is approved - can you change the status of this PR from `draft` to `ready for review` after merging in #45?

**Review by skanwal — APPROVED** [review]
Thanks @alexiswl - looking forward to testing it out for the next RNAsum release.

**Pull request merged**
